migrations/20200901-database/update.sql

```diff
@@ -0,0 +1,2405 @@
+# ************************************************************
```

Comment:
Ik vermoed dat dit een test is, en dus weg gaat uit de definitieve PR?

Reply:
Ja, is weg